__tests__/app.test.js

```diff
@@ -69,3 +69,42 @@ describe("GET /api/articles/:article_id", () => {
       });
   });
 });
+
+describe.only("PATCH /api/articles/:article_id", () => {
+  test("Status:200 request body accepts an object with a key/value pair which holds the number to increment the votes property and returns the updated object", () => {
+    const requestObj = { inc_votes: 1 };
+    return request(app)
+      .patch("/api/articles/1")
+      .send(requestObj)
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.article).toMatchObject({
+          article_id: 1,
+          title: expect.any(String),
+          topic: expect.any(String),
+          author: expect.any(String),
+          body: expect.any(String),
+          created_at: expect.any(String),
+          votes: 101,
+        });
+      });
+  });
+  test("status:404 sends an appropriate and error message when given a valid but non-existent id", () => {
+    const requestObj = { inc_votes: 1 };
+    return request(app)
+      .patch("/api/articles/200")
+      .send(requestObj)
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Invalid article Id");
+      });
+  });
+  test("Status:400 sends an appropriate and error message when given an invalid id", () => {
+    return request(app)
+      .get("/api/articles/invalid_id")
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Bad request");
+      });
+  });
+});
```

Comment:
What if the request object is empty? Or what if it's there but the value in the object isn't a number e.g { inc_votes: notANumber }